www/js/chromecast.js

```diff
@@ -128,7 +132,7 @@ function receiverMessage(namespace, message) {
  * session listener during initialization
  */
 function sessionListener(e) {
-  appendMessage('New session ID:' + e.sessionId);
+  appendMessage(i18n.t('CHROMECAST.NEW_SESSION_ID') + e.sessionId);
```

Comment:
anything going to appendMessage doesn't need to be translated. console logs don't need to be translated.